Refactor TransferWalletBalance

MariaHCD opened the pull request:
Related Web PR: https://github.com/Expensify/Web-Expensify/pull/34127

### Details
<!-- Explanation of the change or anything fishy that is going on -->

### Fixed Issues
<!---
Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing.
Do NOT add the special GH keywords like `fixed` etc, we have our own process of managing the flow.
It MUST be an entire link to the issue; otherwise, the linking will not work as expected.

Make sure this section looks similar to this (you can link multiple issues using the same formatting, just add a new line):

$ https://github.com/Expensify/App/issues/<number-of-the-issue>

Do NOT only link the issue number like this: $ #<number-of-the-issue>
--->
$ https://github.com/Expensify/Expensify/issues/216150

### Tests
<!---
Add a numbered list of manual tests you performed that validates your changes work on all platforms, and that there are no regressions present.
Add any additional test steps if test steps are unique to a particular platform.
Manual test steps should be written so that your reviewer can repeat and verify one or more expected outcomes in the development environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->

Pre-requisite accounts (via clitools.sh `generator:account` and `validator:wallet`)
1. Account A set up with a gold wallet and a validated bank account
2. Account B set up with a gold wallet and a validated bank account

Tests:
1. As Account A send money to Account B
2. As Account B, navigate to Settings > Payments > Transfer Balance

![Screen Shot 2022-07-08 at 9 37 51 PM](https://user-images.githubusercontent.com/12268372/178047243-f6b1be3e-cf85-49cf-84d3-eda4e9d58518.png)

![Screen Shot 2022-07-08 at 9 37 57 PM](https://user-images.githubusercontent.com/12268372/178047266-7aa270a5-bf52-4d6e-8a14-7483b74630f1.png)

3. Click on `Transfer Balance` and then the `Transfer $<amount>` button

![Screen Shot 2022-07-08 at 9 38 53 PM](https://user-images.githubusercontent.com/12268372/178047308-00863f0b-2abe-40a9-87d5-4cc24401c421.png)

4. Verify the transfer completes successfully and you see the success page

![Screen Shot 2022-07-11 at 1 28 08 PM](https://user-images.githubusercontent.com/12268372/178234196-2a8033fd-659e-409a-b433-56bec36f0f62.png)

5. Verify clicking `Done` or the close button directs you back to the Settings > Payments page

![Screen Shot 2022-07-08 at 10 11 44 PM](https://user-images.githubusercontent.com/12268372/178047582-cad7a601-4411-4d4b-8b3e-60db068dd02e.png)

6. Verify the `Transfer Balance` button on the Payments page is disabled when offline

![Screen Shot 2022-07-21 at 12 14 19 PM](https://user-images.githubusercontent.com/12268372/180165371-5ebda572-9300-487d-9401-9000e68ba39b.png)

7. Verify that the button on the Transfer Balance page is disabled when offline

![Screen Shot 2022-07-21 at 12 14 41 PM](https://user-images.githubusercontent.com/12268372/180165362-e9ba6e6a-fa6f-4b27-baae-9de4ee6a8da4.png)

### PR Review Checklist
<!--
This is a checklist for PR authors & reviewers. Please make sure to complete all tasks and check them off once you do, or else Expensify has the right not to merge your PR!
-->
#### Contributor (PR Author) Checklist
- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [ ] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I ran the tests on **all platforms** & verified they passed on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
    - [x] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] Any functional components have the `displayName` property
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose and it is
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn’t already exist
    - [x] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.


<details>
<summary><h4>PR Reviewer Checklist</h4></summary>

- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained “why” the code was doing something instead of only explaining “what” the code was doing.
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [ ] I verified any copy / text that was added to the app is correct English and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named “index.js”. All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] Any functional components have the `displayName` property
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose and it is broken down into smaller components in order to separate concerns and functions
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn’t already exist
    - [ ] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.

</details>

### QA Steps
<!---
Add a numbered list of manual tests that can be performed by our QA engineers on the staging environment to validate that your changes work on all platforms, and that there are no regressions present.
Add any additional QA steps if test steps are unique to a particular platform.
Manual test steps should be written so that the QA engineer can repeat and verify one or more expected outcomes in the staging environment.

For example:
1. Click on the text input to bring it into focus
7. Upload an image via copy paste
9. Verify a modal appears displaying a preview of that image
--->
Pre-requisite accounts:
1. Account A on the wallet beta, set up with a gold wallet and a validated bank account
2. Account B on the wallet beta, set up with a gold wallet and a validated bank account

Tests:
1. As Account A send money to Account B
2. As Account B, navigate to Settings > Payments > Transfer Balance
3. Click on `Transfer Balance` and then the `Transfer $<amount>` button
4. Verify the transfer completes successfully and you see the success page
5. Verify clicking `Done` or the close button directs you back to the Settings > Payments page
6. Verify the `Transfer Balance` button on the Payments page is disabled when offline

![Screen Shot 2022-07-21 at 12 14 19 PM](https://user-images.githubusercontent.com/12268372/180165371-5ebda572-9300-487d-9401-9000e68ba39b.png)

7. Verify that the button on the Transfer Balance page is disabled when offline

![Screen Shot 2022-07-21 at 12 14 41 PM](https://user-images.githubusercontent.com/12268372/180165362-e9ba6e6a-fa6f-4b27-baae-9de4ee6a8da4.png)

- [ ] Verify that no errors appear in the JS console

### Screenshots
<!-- Add screenshots for all platforms tested. Pull requests won't be merged unless the screenshots show the app was tested on all platforms.-->

#### Web

https://user-images.githubusercontent.com/12268372/178234139-06a7a36f-4c48-4fb8-b317-faad70afa512.mp4

#### Desktop

https://user-images.githubusercontent.com/12268372/178480028-3c14d78d-e063-4386-b015-1228cff91da0.mp4

MariaHCD committed:
Use new API command: TransferBalance

MariaHCD committed:
Remove unnecessary saveWalletTransferAmount method

MariaHCD committed:
Remove usage of deprecated API command

MariaHCD committed:
Fix JSDoc and remove unused import

MariaHCD committed:
Add successful balance transfer page

MariaHCD committed:
Fixes for successful transfer balance page

MariaHCD committed:
Show transfer success page

MariaHCD committed:
Add showShouldTransferSuccess Onyx prop

MariaHCD committed:
Import routes

MariaHCD committed:
Add close button

MariaHCD committed:
Allow success and failure data to be set for refactored commands

MariaHCD committed:
Style fixes

MariaHCD committed:
Rename api command

MariaHCD committed:
Update success transfer message

MariaHCD committed:
Update success message

MariaHCD committed:
Remove unused confirm modal

MariaHCD committed:
Delete successful transfer balance page

MariaHCD committed:
Style fixes

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-refactor-transfer-wallet-balance

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-refactor-transfer-wallet-balance

MariaHCD committed:
Rename method

MariaHCD committed:
Specify that we want the API to return onyxData

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-refactor-transfer-wallet-balance

# Conflicts:
#	src/libs/Middleware/SaveResponseInOnyx.js

MariaHCD committed:
Update transfer balance success screen

MariaHCD committed:
Update success text

MariaHCD committed:
Remove unused text and translations

MariaHCD committed:
Fix capitalization

MariaHCD committed:
Add Spanish translations

shawnborton commented on src/pages/settings/Payments/TransferBalancePage.js:
```diff
@@ -169,6 +171,43 @@ class TransferBalancePage extends React.Component {
+                        />
+                        <View style={[styles.pageWrapper, styles.flex1, styles.flexColumn, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                            <Icon
+                                src={Illustrations.TadaYellow}
```
I left this in the other issue, but here is a blue version in case you want to switch it out: [tada--blue.svg.zip](https://github.com/Expensify/App/files/9075044/tada--blue.svg.zip)

MariaHCD replied:
Thanks, Updated!

![Screen Shot 2022-07-11 at 1 28 08 PM](https://user-images.githubusercontent.com/12268372/178235220-ec067568-01e6-492a-88af-e1bcec9b409a.png)

shawnborton replied:
Looks good!

MariaHCD committed:
Add TadaBlue illustration

MariaHCD committed:
Use blue tada icon

thienlnam reviewed (changes requested):
Looking good! Just a few comments

MariaHCD commented on src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js:
```diff
@@ -420,19 +419,6 @@ class BasePaymentsPage extends React.Component {
                         shouldShowCancelButton
                         danger
                     />
-                    <ConfirmModal
```
Yep, removing the confirm modal is intentional, we used to show it after a successful transfer:

<img width="575" alt="Screen Shot 2022-06-16 at 1 43 36 PM" src="https://user-images.githubusercontent.com/12268372/178473248-2013c6ac-0a65-45b6-9e71-231f2895c6d0.png">

But we decided to make it consistent with other success screens instead. Yeah, I think `walletTransfer.transferAmount` isn't used anywhere else. `shouldShowConfirmModal` and `dismissWalletConfirmModal` aren't used either. Updated!

MariaHCD committed:
Remove unused prop `transferAmount`

MariaHCD committed:
Remove unused `shouldShowConfirmModal` and `dismissWalletConfirmModal`

MariaHCD committed:
Rename to `shouldShowSuccess`

JmillsExpensify commented:
Shall we remove the hold now that Web-E PR is merged?

stitesExpensify commented:
@JmillsExpensify we need to wait until the web PR is deployed to production to prevent this from getting merged when it won't work yet :)

thienlnam commented on src/libs/actions/PaymentMethods.js:
```diff
@@ -209,31 +210,53 @@ function clearDebitCardFormErrorAndSubmit() {
+                onyxMethod: 'merge',
+                key: ONYXKEYS.WALLET_TRANSFER,
+                value: {
+                    transferAmount,
```
Do we still use the transferAmount here as well?

MariaHCD replied:
Thanks for catching that! Removed.

ctkochan22 commented:
Should be off hold now?

MariaHCD committed:
Remove unused `transferAmount` prop

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-refactor-transfer-wallet-balance

MariaHCD commented:
Updated and off hold!

stitesExpensify commented on src/pages/settings/Payments/TransferBalancePage.js:
```diff
@@ -169,6 +163,43 @@ class TransferBalancePage extends React.Component {
+                                    {this.props.translate('transferAmountPage.transferSuccess')}
+                                </Text>
+                                <Text style={[styles.mt3, styles.textAlignCenter]}>
+                                    {this.props.translate('transferAmountPage.transferDetail')}
```
We don't want to always show this right? Transferring to a debit card is instant

MariaHCD replied:
That's a good point, re-opening the mockup issue to see if we can get another copy in the case of debit card transfers: https://github.com/Expensify/Expensify/issues/217387

MariaHCD replied:
Just waiting on the Spanish translation and then we should be good to go!

stitesExpensify reviewed (changes requested):
Code LGTM, just one small change!

MariaHCD committed:
Add copy for debit card transfer success

MariaHCD committed:
Show success message based on payment method type

MariaHCD committed:
Style fix

MariaHCD committed:
Add translation

MariaHCD committed:
Disable transfer balance if offline

MariaHCD committed:
Disable transfer button if offline

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-refactor-transfer-wallet-balance

MariaHCD reviewed (commented):
Updated with the correct copy and translations for the success screen. All good for re-review!

Also, disabled the transfer buttons when offline. Although it looks like the alignment of `You appear to be offline` is a little off (which can be fixed in a follow up PR):

![Screen Shot 2022-07-18 at 12 12 51 PM](https://user-images.githubusercontent.com/12268372/179470437-53abbea2-babf-4a84-ac57-12a70b63988f.png)

![Screen Shot 2022-07-18 at 12 13 08 PM](https://user-images.githubusercontent.com/12268372/179470438-d2e7076a-8e6c-4abe-b1bb-2345661eb468.png)

shawnborton commented:
I agree that the alignment is off. I commented in [Slack here](https://expensify.slack.com/archives/C02HWMSMZEC/p1658162822563989) about that.

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-refactor-transfer-wallet-balance

MariaHCD committed:
Add containerStyles prop to OfflineIndicator

MariaHCD committed:
Add OfflineIndicator to BasePaymentsPage and TransferBalancePage

MariaHCD commented:
Merged https://github.com/Expensify/App/pull/9959 into this branch and added the OfflineIndicator to:

 1. BasePaymentsPage:

![Screen Shot 2022-07-19 at 11 31 55 AM](https://user-images.githubusercontent.com/12268372/179695005-4ed4a6be-7b61-416c-b2ae-9309b1bcfacc.png)

2. TransferBalancePage: I think the padding on the bottom is a little too much but that's set on the FixedFooter component and I'm not sure how to adjust that without affecting the styles of the footer when online.

![Screen Shot 2022-07-19 at 11 39 44 AM](https://user-images.githubusercontent.com/12268372/179695017-a14c1311-3cc3-4376-a7b5-808e90025434.png)

Let me know what you think, @shawnborton @ctkochan22

shawnborton commented:
In terms of the offline indicator itself, I think we want it to be something like this:
<img width="1100" alt="image" src="https://user-images.githubusercontent.com/2319350/179780746-7c7ad68e-546b-4c30-9cce-cb6c5d2d796d.png">


Where there is 20px from the left edge to the icon, and below there should be 12px of spacing. Between the icon and the text, there should be a 12px gap. 

Not sure if that's what's going on here or if you are even making those changes here, but just wanted to call it out.

MariaHCD committed:
Adjust icon and styles for OfflineIndicator

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-refactor-transfer-wallet-balance

# Conflicts:
#	src/libs/actions/PaymentMethods.js

MariaHCD committed:
Fix styling of OfflineIndicator on TransferBalancePage

MariaHCD commented:
Updated the styles for the OfflineIndicator, specifically:
- Updated the icon and altered the gap between the icon and text to be 12px
- On the Payments page and on the TransferBalance page, there is 20px from the left edge to the icon, and below there is 12px of spacing

Relevant commits: 
- https://github.com/Expensify/App/pull/9589/commits/1796df02ea1f4a914219e0466cd04cc9f0297894
- https://github.com/Expensify/App/pull/9589/commits/7909bbbe8aef4637f5c96276bfceafe99762b6a5

![Screen Shot 2022-07-20 at 3 24 34 PM](https://user-images.githubusercontent.com/12268372/179982808-8efd8b32-d0bb-48cc-8311-bfe0c1904445.png)

![Screen Shot 2022-07-20 at 4 32 57 PM](https://user-images.githubusercontent.com/12268372/179982817-42916551-55a1-48ef-9ab4-1dfaa19572b5.png)

shawnborton commented on src/components/OfflineIndicator.js:
```diff
@@ -14,33 +15,41 @@ const propTypes = {
             <Icon
-                src={Expensicons.Offline}
+                src={Expensicons.OfflineCloud}
                 width={variables.iconSizeExtraSmall}
```
@MariaHCD Just wanted to confirm, this icon should be shown at 16x16. I think that's the case from seeing the mockups, but just double checking!

MariaHCD replied:
It was set at 12x12 but I've updated to 16x16, thanks!

MariaHCD replied:
![Screen Shot 2022-07-21 at 12 14 19 PM](https://user-images.githubusercontent.com/12268372/180165371-5ebda572-9300-487d-9401-9000e68ba39b.png)

![Screen Shot 2022-07-21 at 12 14 41 PM](https://user-images.githubusercontent.com/12268372/180165362-e9ba6e6a-fa6f-4b27-baae-9de4ee6a8da4.png)

MariaHCD committed:
Fix icon size for OfflineIndicator

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-refactor-transfer-wallet-balance

# Conflicts:
#	src/components/OfflineIndicator.js
#	src/pages/home/report/ReportActionCompose.js
#	src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js
#	src/pages/settings/Payments/TransferBalancePage.js

MariaHCD committed:
Fix errors from merge conflicts

MariaHCD committed:
Optionally override default styles for OfflineIndicator

MariaHCD commented:
Fixed merge conflicts and tweaked the styles for the OfflineIndicator

thienlnam reviewed (dismissed):
Approved but pending another merge conflic

ctkochan22 reviewed (dismissed):
Looks good. Just conflicts

Commit:
fix conflicts

Commit:
use new prop

ctkochan22 commented:
@MariaHCD I hope its okay, I feel like we have all the approves, but cause of the timezone, we are getting approvals after changes that causes conflicts. I just went in and fixed the conflicts

ctkochan22 commented:
@stitesExpensify @shawnborton @thienlnam Can we get a quick re-reveiw, I just took care of the small merge conflict

thienlnam commented:
Just going to merge this before we get any more conflicts - all previous reviewers have approved pending conflicts

OSBotify commented:
:hand: This PR was not deployed to staging yet because QA is ongoing. It will be automatically deployed to staging after the next production release.

MariaHCD commented:
Thanks for moving this forward!

mvtglobally commented:
@MariaHCD @JmillsExpensify @thienlnam Can you QA this internally? We are not able to test wallet due to PROD KYC wall

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/2733895651) to staging by @thienlnam in version: 1.1.86-0 🚀

 platform | result 
 ---|--- 
🤖 android 🤖|success ✅ 
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅ 
🕸 web 🕸|success ✅

MariaHCD commented:
> @MariaHCD @JmillsExpensify @thienlnam Can you QA this internally? We are not able to test wallet due to PROD KYC wall

Asked here! https://expensify.slack.com/archives/C02HWMSMZEC/p1658825528148109?thread_ts=1657045889.543309&cid=C02HWMSMZEC

luacmartins commented:
@MariaHCD did we manage to test this on staging?

yuwenmemon commented:
Tested on staging! It's a pass 🎉

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/2778068298) to production by @yuwenmemon in version: 1.1.86-5 🚀

 platform | result 
 ---|--- 
🤖 android 🤖|success ✅ 
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅ 
🕸 web 🕸|success ✅
